src/noui.cpp

```diff
@@ -14,6 +14,8 @@
 
 #include <boost/signals2/connection.hpp>
 
+bool g_suppress_noui_stderr_output = false;
```

Comment:
wouldn't it be better to not connect this signal at all?

Reply:
I'm likely missing something but if skipping the `uiInterface.ThreadSafeMessageBox_connect` call then `boost::signals2::no_slots_error` will get thrown? What am I doing wrong? :-)

Reply:
Couldn't you connect a `[](){return true;}`?